daemons/gptp/common/avbts_osipc.hpp

```diff
@@ -85,7 +85,8 @@ class OS_IPC {
 		uint32_t sync_count,
 		uint32_t pdelay_count,
 		PortState port_state,
-		bool asCapable ) = 0;
+		bool asCapable,
+      uint64_t mean_path_delay ) = 0;
```

Comment:
Take a pass to fix alignment issues throughout all the files.

Reply:
Done

Reply:
Actually done now